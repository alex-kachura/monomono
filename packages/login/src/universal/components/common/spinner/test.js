import React from 'react';
import Spinner from './';

describe('[Component: Spinner]', () => {
  let component;

  const mockProps = {
    mobileStyles: 'mobile-styles',
    desktopStyles: 'desktop-styles',
  };

  beforeEach(() => {
    component = global.contextualShallow(<Spinner {...mockProps} />);
  });

  it('should render correctly', () => {
    expect(component).toMatchSnapshot();
  });
});
