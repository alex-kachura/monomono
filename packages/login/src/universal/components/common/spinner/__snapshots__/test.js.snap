// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Component: Spinner] should render correctly 1`] = `
<styled.div>
  <styled.div
    desktopStyles="desktop-styles"
    mobileStyles="mobile-styles"
  />
</styled.div>
`;
